mlrun/model_monitoring/db/stores/base/store.py

```diff
@@ -143,3 +143,41 @@ def get_endpoint_real_time_metrics(
         """
 
         pass
+
+    @abstractmethod
+    def write_application_result(self, event: dict[str, typing.Any]):
+        """
+        Write a new application result event in the target table.
+
+        :param event: An event dictionary that represents the application result, should be corresponded to the
+                      schema defined in the :py:class:`~mlrun.common.schemas.model_monitoring.constants.WriterEvent`
+                      object.
+        """
+        pass
+
+    @abstractmethod
+    def get_last_analyzed(self, endpoint_id: str, application_name: str) -> int:
```

Comment:
As discussed : In the sql this function creates new record with null but in the v3io it doesnt, I think that it should do the same.
I suggest that the update method would only update and to write also a `create_last_analyzed_record` method, that we will call from the controller in case the record is not created yet (Also I think it is better to raise an error in case we don't have a record as I wrote in another comment).

Reply:
fixed following our talk - https://github.com/mlrun/mlrun/commit/ecae9118c61db11c9f124b552d65402e254a63e9 
don't think it's required to create a separate API for `create_last_analyzed_record` because anyway we only doing that once through the SQL update process (V3IO KV call put which updates/creates in the same call)